src/cf-forms/src/organisms/form.less

```diff
@@ -0,0 +1,16 @@
+.o-form {
+    &_group {
+        margin-bottom: 30px;
+    }
+
+    &_fieldset {
+        // Overwrite Normalize
```

Comment:
Should we create our own normalize that just has what we want/need? There's also a style on `type=search` inputs that sets the `box-sizing` to `content-box` which has to be overridden with the type attr to get past the specificity.